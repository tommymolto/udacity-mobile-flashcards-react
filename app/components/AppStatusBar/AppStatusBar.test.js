/* global describe, it, expect */

import React from 'react';
import renderer from 'react-test-renderer';
import AppStatusBar from './AppStatusBar';

describe('AppStatusBar', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<AppStatusBar backgroundColor="#292477" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
