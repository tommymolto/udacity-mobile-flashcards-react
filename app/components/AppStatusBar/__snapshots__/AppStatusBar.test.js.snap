// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppStatusBar renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#292477",
      "height": 44,
    }
  }
/>
`;
